packages/components/src/ItemList.test.tsx

```diff
@@ -45,44 +45,66 @@ it('mounts and unmounts properly', () => {
 });
 
 describe('mouse', () => {
-  function clickItem(itemIndex: number, options = {}): void {
+  async function clickItem(
+    user: ReturnType<typeof userEvent.setup>,
+    itemIndex: number,
+    options: {
+      ctrlKey?: boolean;
+      shiftKey?: boolean;
+      dblClick?: boolean;
+    } = {}
```

Comment:
These options get used in a few places, may want to declare as a type in `TestUtils` to refactor it a bit.